kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -48,7 +48,7 @@ object Bench:
 
     abstract class Base[A](expectedResult: A) extends Bench[A](expectedResult):
         def zioBench(): zio.UIO[A]
-        def kyoBenchFiber(): kyo.<[A, kyo.Async] = kyoBench()
+        def kyoBenchFiber(): kyo.<[A, kyo.Async & kyo.Abort[Throwable]] = kyoBench()
```

Comment:
Allow `Abort` in benchmarks since channels and meters can now about with `Closed`.